### PR TITLE
Support turning a list of cue points into a TextTrack

### PR DESCRIPTION
**Pull request by forbesjo**
This PR is to support a list of cue points as a property of a playlist item.

### CODE REVIEW

File: src/playitem.js [hunk 1]
```diff
@@ -1,16 +1,26 @@
...
+    i = (remoteTT && remoteTT.length) || 0,
+    j = (ttl && ttl.length) || 0;
 
   while (i--) {
     player.removeRemoteTextTrack(remoteTT[i]);
   }
+
+
+  while (j--) {
+    ttl.removeTrack_(ttl[j]);
```
Comment (gkatsev) [inline src/playitem.js]: This is not safe and we shouldn't do it. At best, we should set the `mode` of these to `disabled`.

File: src/playitem.js [hunk 2]
```diff
@@ -24,6 +26,15 @@ var playItem = function(player, autoadvanceTimeout, obj) {
     player.addRemoteTextTrack(obj.textTracks[i]);
   }
 
+  if (obj.cuePoints && obj.cuePoints.length) {
+    track = player.addRemoteTextTrack({kind: 'metadata'});
```
Comment (gkatsev) [inline src/playitem.js]: `addRemoteTextTrack` returns a `HTMLTrackElement`, so, you want to set this variable to `trackEl`;
Then `track` should be set to `trackEl.track`.

**Comment by forbesjo** [comment]
@gkatsev wondering if you could take a look at this again.

**Comment by gkatsev** [comment]
LGTM

**Pull request merged**